docs/topics/request-response.rst

```diff
@@ -24,7 +24,7 @@ below in :ref:`topics-request-response-ref-request-subclasses` and
 Request objects
 ===============
 
-.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback, flags])
+.. class:: scrapy.Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback, flags])
```

Comment:
This will break links like:

    :class:`scrapy.http.Request`

In a massive pull request which I am now splitting into smaller, separate pull requests, I dealt with this issue by [having all import paths covered](https://github.com/scrapy/scrapy/pull/3532/files#diff-6ef592ca1364869170edefc0fec563aeR8).